targets/kde

```diff
@@ -9,7 +9,12 @@ CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal kde-plasma-desktop -- network-manager
+if release -lt vivid then
+    PACKNAME='kde-plasma-desktop'
```

Comment:
Shouldn't we use not capital letters for the variable name PACKNAME since it is not global and also we don't do it https://github.com/dnschneid/crouton/blob/master/targets/xorg#L180 here?